aybapi/utils/fetch.py

```diff
@@ -0,0 +1,6 @@
+import requests
+
+
+def fetch(url):
+    res = requests.get(url)
+    return res.json()
```

Comment:
and if it returns a 404, or a non json response (e.g if the server is down and you get a standard whitelabel html response instead, which some providers like cloudflare do)? Do you want to handle that rather than getting a TypeError or whatever, perhaps?